Apto/Plugins/PartsListElement/Client/backend/apto-plugin-parts-list-element/parts-list-element.js

```diff
@@ -0,0 +1,9 @@
+import angular from "angular";
+import Components from './parts-list-element-components';
+
+const AptoBackendModule = angular.module('AptoBackend');
```

Comment:
we should always use apto-loader to register components to the angularjs application, otherwise it would not be overridable, the hint element also do it wrong at the backend. you can see an example at the hint element frontend module